alerting/src/test/kotlin/org/opensearch/alerting/resthandler/MonitorRestApiIT.kt

```diff
@@ -1175,8 +1175,11 @@ class MonitorRestApiIT : AlertingRestTestCase() {
         try {
             client().makeRequest("POST", ALERTING_BASE_URI, emptyMap(), monitor.toHttpEntity())
             fail("Monitor with illegal trigger should be rejected.")
-        } catch (e: java.lang.Exception) {
-            assertNotEquals(e.message, e.message, "Illegal trigger type, ${QueryLevelTrigger::javaClass.name}, for bucket level monitor")
+        } catch (e: java.lang.IllegalArgumentException) {
```

Comment:
Import `IllegalArgumentException`, so you can put `IllegalArgumentException` instead of `java.lang.IllegalArgumentException` here.
Please fix this in the other places as well.